src/utils/registry.ts

```diff
@@ -173,7 +173,7 @@ const DOWNLOAD_URLS = {
   },
 };
 
-export const hostPlatform = ((): BrowserPlatform => {
+function lazyHostPlatform(): BrowserPlatform {
```

Comment:
```suggestion
function hostPlatform(): BrowserPlatform {
```
I would not call it lazy, since it doesn't contain any logic to load it e.g. asynchronously or store its value somewhere. Normal `hostPlatform` should be fine. Ah saw you are caching it later on, but then this file has nothing to do with cache.